recipes/libpcap/all/conanfile.py

```diff
@@ -29,11 +29,13 @@ class LibPcapConan(ConanFile):
         "shared": [True, False],
         "fPIC": [True, False],
         "enable_libusb": [True, False],
+        "with_snf": [None, True, False],
```

Comment:
```suggestion
        "with_snf": [True, False],
```

Let's keep binary decision, no auto values. We can sure about its default value: https://github.com/the-tcpdump-group/libpcap/blob/master/configure.ac#L1515